tests/circuitpython/environ_test.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+try:
+    get_key
```

Comment:
Is this the new name for Unix only? Maybe a clearer name for this is `getenv_from_file`?